Posit Infrastructure/FAQs.md

```diff
@@ -115,12 +116,17 @@ install.packages("ranger")
 Some packages, e.g. `{xlsx}`, `{XLconnect}`, depend on an installation of [Java](https://en.wikipedia.org/wiki/Java_(software_platform)) in order to work.  There are no current or future plans to support Java on Posit Workbench.  Alternative packages such as `{openxlsx}` that do not rely on Java should be used instead.
 
 #### What should I do if I encounter a 'failed to lock directory' error?
+
```

Comment:
Added a clear line here deliberately as I noticed I'd forgotten to do so in a previous commit.